SUMMARY.md

```diff
@@ -129,37 +129,36 @@
   * [Self Service](operations/business-operations/self-service.md)
   * [Salesforce Administration](operations/business-operations/salesforce-administration.md)
 * [Finance](operations/finance/README.md)
-  * [Spending Company Money](operations/finance/spending-company-money/README.md)
+  * [Budget](operations/finance/spending-company-money/budget.md)
+  * [Purchasing](operations/finance/purchasing/README.md)
+    * [Purchase Submission Request](company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md)
+    * [Virtual Credit Card](company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md)
+    * [Contracts and Signing](company/how-to-guides-for-staff/how-to-purchase/contracts-and-signing.md)
     * [How to Use Accrualify](operations/finance/spending-company-money/how-to-use-accrualify/README.md)
       * [Setting up your Account](operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md)
       * [Navigating Accrualify](operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md)
       * [Submitting a Purchase Order](operations/finance/spending-company-money/how-to-use-accrualify/how-to-submit-a-po.md)
       * [Reviewing a Purchase Order](operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-po.md)
-      * [Reviewing a Vendor Invoice](operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md)
-      * [Frequently Asked Questions](operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md)
-    * [Budget](operations/finance/spending-company-money/budget.md)
-    * [Procurement](operations/finance/spending-company-money/procurement.md)
-      * [Purchase Submission Request](/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md)
-      * [Virtual Credit Card](company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md)
-      * [Vendor Onboarding](company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/README.md)
-      * [Vendor Invoice Submission](company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-invoice-mattermost.md)
-  * [Arrange a Bounty Program](/company/how-to-guides-for-staff/how-to-purchase/how-to-arrange-a-bounty-program.md)
+  * [Onboarding](operations/finance/onboarding/README.md)
+    * [Vendor Onboarding](company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/README.md)
+    * [ROW Staff Onboarding](company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/row-onboarding.md)
```

Comment:
After "ROW Staff Onboarding", we put an indicator (not sure how this can look), that says "How-to Guides" so that the person looking through this section understands that the "Accrualify Vendor Portal Guide", "Reviewing a vendor invoice" and "Frequently asked questions" is is related to how to use Accrualify?